### PR TITLE
Add zenodo file for release publication

### PR DESCRIPTION
**Pull request by lwasser**
hey @arianesasso @NickleDave @batalex can you please have a look at this zenodo file and make sure your info is correct / add orcid id's  and organizations if you have them and want to?

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "creators": [
+      {
+        "affiliation": "pyOpenSci Lab",
```
Comment (lwasser) [inline .zenodo.json:4]:
```suggestion
        "affiliation": "pyOpenSci",
```

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      },
+      {
+        "affiliation": "Earth Lab, University of Colorado - Boulder",
+        "name": "Sasso, Ariane",
+        "orcid": ""
+      },
+      {
+        "affiliation": "",
+        "name": "Batisse , Alexandre",
+        "orcid": ""
```
Comment (Batalex) [inline .zenodo.json:21]:
```suggestion
        "name": "Batisse, Alexandre",
        "orcid": "0000-0001-7912-3422"
```

Thanks for the entry; this seems well paid for just a few comments 😁

**Review by NickleDave — APPROVED** [review]
👍 LGTM! I'll just be unaffiliated for now 🙂

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        "name": "Wasser, Leah",
+        "orcid": "0000-0002-8177-6550"
+      },
+      {
+        "affiliation": "",
+        "name": "Nicholson, David",
+        "orcid": "0000-0002-4261-4719"
+      },
+      {
+        "affiliation": "Earth Lab, University of Colorado - Boulder",
```
Comment (arianesasso) [inline .zenodo.json:14]:
```suggestion
        "affiliation": "Hasso Plattner Institute",
```

File: .zenodo.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        "name": "Sasso, Ariane",
+        "orcid": ""
```
Comment (arianesasso) [inline .zenodo.json:16]:
```suggestion
        "orcid": "0000-0002-3669-4599"
```

**Comment by lwasser** [comment]
Thank you @arianesasso @NickleDave @Batalex ! i really appreciate your help with the guides. @Batalex i consider all contributions even feedback / reviews to be significant. as you're taking time to read through and in turn guide the direction of pyOpenSci. So I appreciate the effort and time! And I appreciate all of you being here.

**Pull request merged**